app/controllers/calendar_controller.rb

```diff
@@ -21,6 +21,7 @@ def index
   def create_calendar_props
     @props = {conferenceRooms: ConferenceRoom.all,
               initialEvents: events,
+              userEmail: session[:email],
```

Comment:
Maybe it would be better to use React context for user data instead of passing `userEmail` to all components?